mercury/onchain_config.go

```diff
@@ -26,7 +27,7 @@ var _ mercury.OnchainConfigCodec = StandardOnchainConfigCodec{}
 // returned by EncodeValueInt192.
 type StandardOnchainConfigCodec struct{}
 
-func (StandardOnchainConfigCodec) Decode(b []byte) (mercury.OnchainConfig, error) {
+func (StandardOnchainConfigCodec) Decode(ctx context.Context, b []byte) (mercury.OnchainConfig, error) {
```

Comment:
Again not sure why the codecs need a ctx

Reply:
This call goes over GRPC because the codec is implemented in the chain relayer.